pay-api/src/pay_api/models/eft_credit.py

```diff
@@ -61,11 +61,3 @@ class EFTCredit(BaseModel):  # pylint:disable=too-many-instance-attributes
     def find_by_payment_account_id(cls, payment_account_id: int):
         """Find EFT Credit by payment account id."""
         return cls.query.filter_by(payment_account_id=payment_account_id).all()
-
-    @classmethod
-    def update_account_by_short_name_id(cls, short_name_id: int, payment_account_id: int):
```

Comment:
Not needed anymore, as short name credits can apply to multiple accounts now via new linking table